fog/report/server/src/service.rs

```diff
@@ -69,10 +77,18 @@ impl<E: RecoveryDbError> Error<E> {
 impl<R: ReportDb + Clone + Send + Sync> Service<R> {
     /// Creates a new report service node (but does not create sockets and start
     /// it etc.)
-    pub fn new(report_db: R, materials: Materials, logger: Logger) -> Self {
+    pub fn new(
+        report_db: R,
+        materials: Materials,
+        postgres_retry_count: usize,
+        postgres_retry_millis: u64,
+        logger: Logger,
+    ) -> Self {
```

Comment:
note: I would rather make this `new` function just take the `Config` object than unpack the whole struct, but I decided against making that change right now